WorldCupWrapped/Repositories/TeamRepository/ITeamRepository.cs

```diff
@@ -10,7 +11,6 @@ public interface ITeamRepository: IGenericRepository<Team>
         public Task<List<Team>> GetTeamsByGroupAndPosition(string group, string groupRanking);
 
         public Task<List<Team>> GetAllTeams();
-
-        public Task UpdateTeams();
+        public Task<List<TrophyCount>> GetTeamTrophies(string teamName);
```

Comment:
Am inteles cum ai gandit. Merge bine asa, eu ma gandeam sa facem repo service si controller direct pt TeamTrophies ca practic avem o clasa in plus asa, dar e cam acelasi lucru.

Reply:
era nevoie de un mapper pentru o structura de genul {Trophy: trophy, int: count}, si singura varianta cam era sa fac un TrophyCount